src/components/matrix/Matrix.java

```diff
@@ -0,0 +1,117 @@
+package src.components.matrix;
+
+//Hello Jeremy
+
+public class Matrix {// extends components.standard<Matrix> {
+    private int[][] rep;
+
+    private void createNewRep(int i, int j) {
+        this.rep = new int[i][j];
+
+        for (int x = 0; x < i; x++) {
+            for (int y = 0; y < j; y++) {
+                this.rep[x][y] = 0;
+            }
+        }
+    }
+
+    public Matrix(int[][] m) {
+        this.rep = m.clone();
+    }
+
+    public Matrix(int i, int j) {
+        this.createNewRep(i, j);
+    }
+
+    public int get(int i, int j) {
+        return this.rep[i][j];
+    }
+
+    public void set(int i, int j, int n) {
+        this.rep[i][j] = n;
+    }
+
+    public void clear() {
+        this.createNewRep(this.rep.length, this.rep[0].length);
+    }
+
+    public int[] size() {
```

Comment:
Interesting idea! Is this a pair of length and width? I would expect size to return the total number of elements. Maybe you have a dimensions method that returns this.